src/client/app/containers/InitializationContainer.js

```diff
@@ -1,6 +1,8 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Why did the spacing here get changed? I don't actually care, I'm just curious. Do we have a standard style?